vald/values.yaml

```diff
@@ -653,7 +653,7 @@ indexManager:
   revisionHistoryLimit: 2
   terminationGracePeriodSeconds: 30
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blueviolet.svg?style=flat-square)
@kpango Does this component need HPA?

Reply:
We don't need HPA about this component

Reply:
Okay. Thanks.